tests/test_getters.py

```diff
@@ -33,7 +34,7 @@
 # See if data is available from current location
 # Expect to run from NCSA test stand
 
-# mapper = pathlib.Path("/project/shared/auxTel/_parent/_mapper")
+# FIXME
```

Comment:
In DM we have `TODO` comments, but they have to have a ticket number. Do you allow merges with unticketed `# FIXME` still in?

Reply:
We need tickets as well... except the idea was to fix it before committing as I mentioned it was an issue elsewhere.
Admittedly I think this one can just be removed as it was a discussion I had with Michael.